### PR TITLE
launch: fixes tile padding

### PR DESCRIPTION
**Pull request by tylershuster**
fixes https://github.com/urbit/landscape/issues/129

### CODE REVIEW

**Review by matildepark — CHANGES_REQUESTED** [review]
image

Padding isn't helping here I think.

image

**Comment by tylershuster** [comment]
feckin scrollbar

**Comment by tylershuster** [comment]
Ok, this is a bit bold but I think it achieves what we want without sacrificing usability. The scrollbar will no longer be visible on the home screen but it will still be scrollable. This fixes the padding issues

File: pkg/interface/src/views/apps/launch/app.js [hunk 1]
```diff
@@ -32,10 +41,10 @@ export default class LaunchApp extends React.Component {
         <Helmet>
           <title>OS1 - Home</title>
         </Helmet>
-        <Box height='100%' overflowY='scroll'>
+        <ScrollbarLessBox height='100%' overflowY='scroll'>
```
Comment (matildepark) [inline pkg/interface/src/views/apps/launch/app.js:44]: TBH we're going to want this same thing on the sidebar probably. Separate PR.

**Review by matildepark — APPROVED** [review]

**Pull request merged**